sass/_QuerySuggestPreview.scss

```diff
@@ -0,0 +1,33 @@
+@import 'MagicBox/Colors';
+.coveo-suggestion-container {
+  display: inline-flex;
+  flex-wrap: nowrap;
+  align-items: stretch;
+  border: 1px solid $grey-500;
+  min-width: 100%;
+
+  #coveo-magicbox-suggestions {
+    border: none !important;
```

Comment:
can't you do it without the `!important`?

Reply:
I can try for a bit.

Right now, the border is on this element when the QuerySuggestPreview is not present, so I need to delete the border when the component is present.

Since the selector in the MagicBox is more precise (from what I understood) it will be applied last so if i removed the important the border will be present when the QuerySuggestPreview is present

I will try to see in sass if i can do a `not(QuerySuggestPreviewSelector)` before applying the style